qvisor/src/runc/runtime/vm.rs

```diff
@@ -215,18 +211,10 @@ impl VirtualMachine {
         IOURING.SetValue(sharespace.GetIOUringAddr());
 
         unsafe {
-            KERNEL_PAGETABLE.SetRoot(VMS.read().pageTables.GetRoot());
```

Comment:
What is replacing these lines to set up the kernel pagetable? Perhaps I'm missing something..

Reply:
After refactoring, the host no longer needs the kernel stack, which is used to allocate tasks on private memory.